src/scripts/continue.sh

```diff
@@ -1,24 +1,22 @@
-#!/bin/sh
-
-set -e
```

Comment:
Noticed this appearing in my jobs, no need for a shebang CCI adds them automatically.

```shell
#!/bin/bash -eo pipefail
#!/bin/sh

set -e
...
```

+ it's already set.

Reply:
https://app.circleci.com/pipelines/github/bjd2385/dynamic-continuation-orb/302/workflows/a9000cb6-64ae-4124-8064-29cf26f6e189/jobs/812?invite=true#step-108-1